Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
     .executable(name: "swift-bundler", targets: ["swift-bundler"])
   ],
   dependencies: [
-    .package(url: "https://github.com/stackotter/swift-argument-parser", branch: "main"),
+    .package(url: "https://github.com/Wabi-Studios/stackotter-parser.git", branch: "main"),
```

Comment:
If you PR your argument parser changes against my argument parser repo then we can change this back (but that can happen after this PR, I'm not too fussed)